client/ayon_harmony/api/js/AyonHarmonyAPI.js

```diff
@@ -195,6 +195,35 @@ AyonHarmonyAPI.getNodesNamesByType = function(nodeType) {
 };
 
 
+/**
+ * Create new Composite node in Harmony.
+ * @function
+ * @param {array} args Arguments, see example.
+ * @return {string} Resulting node.
+ *
+ * @example
+ * // arguments are in following order:
+ * var args = [
+ *  nodeName,
+ *  nodeType,
+ *  selection
+ * ];
+ */
+AyonHarmonyAPI.createContainer = function(args) {
```

Comment:
```suggestion
AyonHarmonyAPI.createNodeContainer = function(args) {
```
I think both alternatives must be explicit. As they are two workflow behaviors, there is no default `createContainer`, the appropriate workflow must be decided by the creator plugin.